debian/clearwater-config-manager.install

```diff
@@ -4,6 +4,8 @@ clearwater-config-manager.root/* /
 # Install the etcd plugins
 src/clearwater_etcd_plugins/clearwater_config_manager/shared_config_plugin.py /usr/share/clearwater/clearwater-config-manager/plugins/
 src/clearwater_etcd_plugins/clearwater_config_manager/dns_json_plugin.py /usr/share/clearwater/clearwater-config-manager/plugins/
+src/clearwater_etcd_plugins/clearwater_config_manager/rph_json_plugin.py /usr/share/clearwater/clearwater-config-manager/plugins
```

Comment:
As you noticed, the RPH config is ending up on all node types. To fix this, you'll want to move these lines into sprout's install scripts.

Reply:
moved